@alalazo weird error from mypy:
```
lib/spack/spack/package.py:1272: error: Decorated property not supported
```
is that an error in the package, or an internal limitation in mypy?

--- a/lib/spack/spack/package.py
+++ b/lib/spack/spack/package.py
@@ -1269,7 +1269,7 @@ class PackageBase(six.with_metaclass(PackageMeta, PackageViewMixin, object)):
             raise ValueError("Can only get the arch for concrete package.")
         return spack.architecture.arch_for_spec(self.spec.architecture)
 
-    @property
+    @property  # type: ignore
     @memoized
     def compiler(self):
         """Get the spack.compiler.Compiler object used to build this package"""

